taskvine/src/worker/vine_cache.h

```diff
@@ -30,11 +30,19 @@ typedef enum {
 } vine_cache_type_t;
 
 typedef enum {
+	VINE_FILE_UNKNOWN,
 	VINE_FILE_READY,
 	VINE_FILE_FAILED,
 	VINE_FILE_PROCESSING,
 } vine_file_status_type_t;
 
+typedef enum {
```

Comment:
Seems like this is duplicating `vine_file_status_t` you could just re-use that type.